features/_version.feature

```diff
@@ -0,0 +1,44 @@
+Feature: UA is expected version
+
+    @series.all
+    @uses.config.check_version
+    @uses.config.machine_type.lxd.container
+    @uses.config.machine_type.lxd.vm
+    @uses.config.machine_type.aws.generic
+    @uses.config.machine_type.aws.pro
+    @uses.config.machine_type.azure.generic
+    @uses.config.machine_type.azure.pro
+    @uses.config.machine_type.gcp.generic
+    @uses.config.machine_type.gcp.pro
```

Comment:
My only concern here is that some of the machine types don't support all releases, like the PRO ones that are only supported by LTS releases. However, since we don't have entries for those tests in `tox.ini` (like `behave-gcppro-21.04`), I don't think there is any risk here about them.

So I am fine with this approach.

Reply:
Ahh right - That is a good point that reading this test, it looks like it could run a hirsute gcppro instance, which could be confusing. But like you said, in practice the other releases should be skipped based on how we use tags for each test suite to only run the relevant releases.

So yeah, not perfect, but I think good enough for this release at least.